app/config.go

```diff
@@ -676,6 +680,22 @@ func newDefaultGenesisState() []byte {
 	})
 	res := Codec().MustMarshalJSON(accountGenesis)
 	defaultGenesis[auth.ModuleName] = res
+	// set address as application too
+	rawApps := defaultGenesis[appsTypes.ModuleName]
+	var appsGenesis appsTypes.GenesisState
+	types.ModuleCdc.MustUnmarshalJSON(rawApps, &appsGenesis)
+	appsGenesis.Applications = append(appsGenesis.Applications, appsTypes.Application{
```

Comment:
Should this be in the default genesis?

Reply:
I think so, people would likely want to do exactly what I did to test for relays